src/index.ts

```diff
@@ -66,7 +67,9 @@ export function setup(app: _express.Application, options: SetupOptions) {
 
 	// redirect to https if needed, except for requests to
 	// the /ping endpoint which must always return 200
-	app.use(fixProtocolMiddleware(['/ping']));
+	app.use(
+		fixProtocolMiddleware(['/ping'].concat(options.skipHttpsPaths || [])),
```

Comment:
You don't need `|| []` really -- TS won't let anyone set `skipHttpsPaths` to `null`.

Reply:
Good point, I'll set it to be a ? param, since it shouldn't force any existing code to add the param.